apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -24,6 +23,232 @@ namespace Google.Cloud.Bigtable.V2
     public partial class BigtableClient
     {
         /// <summary>
+        /// Mutates a row atomically based on the output of a predicate Reader filter.
```

Comment:
If the row doesn't exist at all before and falseMutations contains some mutations, is the row created with those mutations?

Reply:
Yes (see `CheckAndMutateRowTest.NullPredicate_MissingRow`). Do you think something should go in the remarks here?